content/documentation/pages/3-stream-developer-guides/2-streams/1-standalone-stream-rabbitmq.md

```diff
@@ -0,0 +1,207 @@
+For instance, testing the `Source` application requires the `output` binding to be set a test `exchange` on RabbitMQ.
+We also need to set the server port to be unique.
+
+```
+spring.cloud.stream.bindings.output.destination=test-usage-detail
+server.port=9090
+```
+
+You can run the standalone `UsageDetailSender` source application as,
+
+```
+java -jar usage-detail-sender-0.0.1-SNAPSHOT.jar --spring.cloud.stream.bindings.output.destination=test-usage-detail --server.port=9090
+```
+
+### Sample Processor
+
+#### Initialzr
+
+1. Visit the [Spring Initialzr site](https://start.spring.io/).
+1. Select the latest release of spring boot.
+1. Create a new Maven project with a Group name of `com.example` and an Artifact name of `usage-cost-processor`.
+1. In the Dependencies text box, type `Rabbitmq` to select the RabbitMQ binder dependency.
+1. In the Dependencies text box, type `Cloud Stream` to select the Spring Cloud Stream dependency.
+1. Click the Generate Project button.
+1. Unzip the usage-cost-processor.zip file and import the project into your favorite IDE.
+
+##### Initialzr Shortcut
+
+Another option instead of using the UI to initialize your project you can do the following:
+
+1. Click the [here](https://start.spring.io/starter.zip?fakeusernameremembered=&fakepasswordremembered=&type=maven-project&language=java&bootVersion=2.1.4.RELEASE&baseDir=usage-cost-processor&groupId=io.spring.dataflow.sample&artifactId=usage-cost-processor&name=usage-cost-processor&description=Sample+project+for+Spring+Cloud+Stream+Processor&packageName=com.example.demosource&packaging=jar&javaVersion=1.8&inputSearch=&style=amqp&style=cloud-stream) to download the preconfigured usage-cost-processor.zip.
+2. Unzip the usage-cost-processor.zip file and import the project into your favorite IDE
+
+#### Biz Logic
+
+1.  In your favorite IDE create the `io.spring.dataflow.sample.domain` package
+1.  Create a `UsageDetail` class in the `io.spring.dataflow.sample.domain` using your favorite IDE that looks like the contents in [UsageDetail.java](https://github.com/spring-cloud/spring-cloud-dataflow-samples/blob/master/dataflow-website/stream-developer-guides/streams/usage-detail-sender/src/main/java/com/example/demo/domain/UsageDetail.java).
+    This `UsageDetail` model contains `userId`, `data` and `duration`.
+1.  Create a `UsageCostDetail` class in the `io.spring.dataflow.sample.domain` using using your favorite IDE that looks like the contents in [UsageCostDetail.java](https://github.com/spring-cloud/spring-cloud-dataflow-samples/blob/master/dataflow-website/stream-developer-guides/streams/usage-cost-processor/src/main/java/com/example/demo/domain/UsageCostDetail.java).
+    This `UsageCostDetail` model contains `userId`, `callCost` and `dataCost`.
+1.  Create the `Processor` application that receives the `UsageDetail` from the previously created `source`, computes the call/data cost and returning the `UsageCostDetail`.
```

Comment:
Its unclear that we want to create a class. May want to reword this to something like:
"In the io.spring.dataflow package create a UsageCostProcessor class using your favorite IDE that looks like the contents below." It's unclear where to create the class as well.